scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -329,7 +329,6 @@ public static BuildTargetResult CompileStage1(BuildTargetContext c)
                 dotnet: DotNetCli.Stage0,
                 outputDir: Dirs.Stage1);
 
-            CleanOutputDir(Path.Combine(Dirs.Stage1, "sdk"));
```

Comment:
Looks like the method `CleanOutputDir` can now be removed as well. And also the `FilesToClean` static array.